docs: note attaching cluster with cert-manager (#4301)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/clusters/attach-cluster/requirements-for-attaching/index.md

Consider the additional resource requirements for running the platform services you want Kommander to manage, and ensure that your existing clusters comply.
-->

> ## Attaching existing clusters that has cert-manager installed

Reviewer: ```suggestion
## Attaching existing clusters with cert-manager installed
```

> If you are attaching clusters that has cert-manager installed, the Kommander provided cert-manager HelmRelease will fail to deploy due to your existing cert-manager installation. As long as the pre-existing cert-manager is functioning as expected, this is safe to ignore.

Reviewer: ```suggestion
If you are attaching clusters that already have cert-manager installed, the cert-manager HelmRelease provided by Kommander will fail to deploy, due to your existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this.
```

> If you are attaching clusters that already have cert-manager installed, the cert-manager HelmRelease provided by Kommander will fail to deploy, due to your existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this.

Reviewer 2: ```suggestion
If you are attaching clusters that already have cert-manager installed, the cert-manager HelmRelease provided by Kommander will fail to deploy, due to the existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this failure, it will have no impact on the operation of the cluster.
```

> If you are attaching clusters that already have cert-manager installed, the cert-manager HelmRelease provided by Kommander will fail to deploy, due to your existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this.

Reviewer 3: nit, mono-spacing the resource type to match the style of the rest of the doc:
```suggestion
If you are attaching clusters that already have cert-manager installed, the cert-manager `HelmRelease` provided by Kommander will fail to deploy, due to your existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this.
```

Author: done

> 

Reviewer 3: Per today's standup, it's probably worth calling out explicitly that a Konvoy cluster will have cert-manager installed, and therefore this will occur. Maybe something like this?

```
<p class="message--note"><strong>NOTE: </strong>Clusters provisioned using Konvoy have cert-manager installed. Kommander's cert-manager `HelmRelease` will fail to deploy on a Konvoy cluster.</p>
```

Author: how does this look?

Reviewer 4: Pushing a change for styles, that's all

Consider the additional resource requirements for running the platform services you want Kommander to manage, and ensure that your existing clusters comply.
-->

## Attach clusters with an existing cert-manager installation

If you are attaching clusters that already have cert-manager installed, the cert-manager `HelmRelease` provided by Kommander will fail to deploy, due to the existing cert-manager installation. As long as the pre-existing cert-manager functions as expected, you can ignore this failure. It will have no impact on the operation of the cluster.

> <p class="message--note"><strong>NOTE: </strong>Clusters provisioned using <code>dkp</code> (Konvoy2 clusters) have cert-manager installed. Kommander's cert-manager <code>HelmRelease</code> will fail to deploy on a Konvoy cluster.</p>

Reviewer 5: Even EKS and AKS clusters are provisioned using DKP. Just say:

DKP Konvoy clusters have cert-manager already installed, hence Kommander's cert-manager will fail..

Otherwise looks good to me. @cwyl02

Author: done
